ui/modal_templates.go

```diff
@@ -59,14 +65,32 @@ type modalLoad struct {
 }
 
 func (win *Window) LoadModalTemplates() *ModalTemplate {
+	editorBoarderColor := color.NRGBA{41, 112, 255, 255}
```

Comment:
Border is spelt wrongly and this colour is already declared in the decredmaterial package